### ISSUE TITLE
Unable to start ./bin/forward binary

### ISSUE THREAD

**Issue by krouziciorel** [TamaMcGlinn/AdaBots_examples#3]
Thank you very much for this interesting project, teaching with Ada is certainly a step in the right direction. I tried following the instructions:

https://adabots.net/installation

to compile AdaBots_examples, for a successful compilation I was forced to modify the line for Source_Dirs in the file adabots_examples.gpr as follows:

for Source_Dirs use ("src", "src/bot_lib", "src/construction", "src/api", "src/small", "src/util"); 

and if I try to run 

./forward

or other binaries, I always see this error:

raised CONSTRAINT_ERROR : a-envvar.adb:208 explicit raise
[./forward]
0x52ecc4 Ada.Environment_Variables.Value at a-envvar.adb:208
0x428ffb Adabots.Create_Dispatcher at adabots.adb:16
0x42922c Adabots.Create_Turtle at adabots.adb:27
0x40df1c Forward at forward.adb:4
0x40ecc8 Main at b__forward.adb:760
[/usr/lib/libc.so.6]
0x7ab42431ecce
0x7ab42431ed88
[./forward]
0x40dda3 _start at ???
0xfffffffffffffffffe

I tried the translation in both Arch and Debian Linux with the same result. Can you please try on your machine?

**Comment by TamaMcGlinn** [comment]
True, sorry about this, it's because I've been modifying things to work using a proxy server hosted on adabots.net, in order to simplify the Adabots client side (so it no longer needs to host its own webserver). It's a step in the right direction, but not quite finished right now. Switching to Adabots version 1.5.0 (in Adabots_examples, that would be [commit 3b55c5ffc5438df51444b2903060b3a0419d6764](https://github.com/TamaMcGlinn/AdaBots_examples/commit/3b55c5ffc5438df51444b2903060b3a0419d6764)) should fix the issue. I will push that as the main branch, and then develop can point at the latest development head for now until these proxy issues are fixed.

**Comment by krouziciorel** [comment]
Dear Tama McGlinn,

Thank you very much for your prompt reply. I'm currently studying Ada from the very good Ada95_Problem_Solving_Program_Design textbook, so I'll definitely wait until the proxy problems are fixed and then try to get Adabots working (given the Alire tool, I would appreciate it if not only Adabots but also Adabots_examples were available here).
